contrib/remote_hsmd/hsmd.c

```diff
@@ -391,6 +396,72 @@ static void workaround_init_bolt12(const struct secret *hsm_secret, struct pubke
 		              "Could derive bolt12 public key.");
 }
 
+static void persist_pubstuff(const struct node_id *node_id,
```

Comment:
s/pubstuff/node_id/ everywhere?

We can probably get the other items from the signer in the future.

Reply:
fixed